combat-trainer.lic

```diff
@@ -177,7 +177,7 @@ class SetupProcess
     if game_state.weapon_skill == 'Targeted Magic' && game_state.weapon_name
       bput("invoke #{game_state.weapon_name}", 'You')
       waitrt?
-    end
+    end 
   end
```

Comment:
You should check if your text editor has an option to remove trailing spaces, can make a mess in big change sets.